app/assets/stylesheets/forms.scss

```diff
@@ -0,0 +1,162 @@
+// Place all the styles related to the form controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+
+// new page styles
+.whole-container{
+  .header{
+    box-shadow: 0 0 50px 0 aliceblue;
+
+    .container{
+      .pagetitle{
+        padding: 1rem 1rem 0;
+      
+        h2{
+          margin: 0;
+        }
+      }
+    }
+    .nav{
```

Comment:
Do not override classes of bootstrap, it'll makes it hard to debug the issues, what you can do is add a new class with all these customizations